lambdas/process-cri-callback/src/test/java/uk/gov/di/ipv/core/processcricallback/pact/passportCri/ContractTest.java

```diff
@@ -151,75 +148,75 @@ class ContractTest {
 
     private static final String FAILED_VC_BODY =
             """
+            {
+              "iss": "dummyPassportComponentId",
+              "sub": "test-subject",
+              "nbf": 4070908800,
+              "exp": 4070909400,
+              "vc": {
+                "@context": [
+                  "https://www.w3.org/2018/credentials/v1",
+                  "https://vocab.london.cloudapps.digital/contexts/identity-v1.jsonld"
+                ],
```

Comment:
Is it worth us adding this context parameter to the other tests as well? Or perhaps it's better to wait for the producer-side to confirm in each case

Reply:
Hmm, I'm not sure where that comes from - it's not documented here: https://govukverify.atlassian.net/wiki/spaces/PYI/pages/3239772254/2.3.1.2.+Passport+Verifiable+Credential+VC so probably easiest to wait for provider side tests and fix it along with any other issues there.